src/readLAS.cpp

```diff
@@ -104,16 +106,16 @@ List lasdatareader(CharacterVector file, bool Intensity, bool ReturnNumber, bool
     // First pass to read the whole file. Aims to know how much memory we must allocate
     if(stream)
     {
-      lasfilter.parse_string(filterchar);
+//       lasfilter.parse_string(filterchar);
 
       npoints = 0;
       while (lasreader->read_point())
       {
-        if(!lasfilter.filter(&lasreader->point))
+//         if(!lasfilter.filter(&lasreader->point))
```

Comment:
It means the reader skip automatically the points based on the filter which is now into the reader. Right?